core/src/main/web/app/controlpanel/controlpanel-directive.js

```diff
@@ -60,6 +63,17 @@
           return bkMenuPluginManager.getMenus();
         };
 
+        if (bkUtils.isElectron){
+          window.addEventListener('focus', function() {
```

Comment:
So what is the correct approach here? Would getting rid of the anonymous function (declare it outside of the if) do it?